lua/orgmode/config/defaults.lua

```diff
@@ -42,6 +42,7 @@ local DefaultConfig = {
   org_edit_src_content_indentation = 0,
   diagnostics = true,
   win_split_mode = 'horizontal',
+  win_border = 'rounded',
```

Comment:
I think we should stick with `single` here since that is currently hardcoded everywhere.

Reply:
done